examples/3d_drawing/plot_3d_animation_walk.py

```diff
@@ -0,0 +1,72 @@
+"""
+====================
+Animation: Node Walk
+====================
+
+Another animation example, illustrating a walk along the nodes as the view
+rotates.
+
+"""
+import numpy as np
+import networkx as nx
+import random
+import matplotlib.pyplot as plt
+from matplotlib import animation
+
+###############################################################################
+# Define a graph to plot.
+# -----------------------
+#
+# Pick a graph for visualization in 3D, such as one of the Platonic solids
+
+G = nx.dodecahedral_graph()
+pos = nx.spring_layout(G, dim=3, seed=2734)
+nodes = np.array([pos[v] for v in G])
+edges = np.array([(pos[u], pos[v]) for u, v in G.edges()])
+
+###############################################################################
+# Random walk on rotating 3D graph animation.
+# -------------------------------------------
+#
+# Modify the plot itself along with the view, showing a wandering node that
+# travels to a new neighbor every 5 frames
+
+
+def init():
+    ax.clear()
+    ax.scatter(*nodes.T, alpha=0.2, s=100, color="blue")
+    for vizedge in edges:
+        ax.plot(*vizedge.T, color="gray")
+    # Intialize the "walking" node
+    ax.plot(*nodes[0], alpha=1, marker="s", color="red")
+    ax.grid(False)
+    ax.set_axis_off()
+    plt.tight_layout()
```

Comment:
This gives me a warning:
UserWarning: The figure layout has changed to tight
  plt.tight_layout()
  
delete, and use 
`fig = plt.figure(layout='tight')` 
below instead

Reply:
The problem with this is that the "tightness" can only be determined after the axis object is added. I went ahead and used `fig.tight_layout()` after the `fig.add_subplot` call, which should be equivalent.